### PR TITLE
add lang attribute to viewed article

### PR DESCRIPTION
**Pull request by narender2031**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

This PR will improve the accessibility for the articles, as per the requirement for the #19849

## Related Tickets & Documents

- Related Issue #19849 

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [ ] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [ ] Checked with [axe DevTools](https://www.deque.com/axe/) and addressed `Critical` and `Serious` issues?
- [ ] Color contrast tested?

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

### CODE REVIEW

File: app/views/articles/show.html.erb [hunk 1]
```diff
@@ -99,8 +100,8 @@
             <a class="crayons-article__cover" href="<%= cloud_cover_url(@article.main_image) %>">
               <img
                 src="<%= cloud_cover_url(@article.main_image) %>"
-                style="aspect-ratio: auto 1000 / <%= @article.main_image_height%>;"
-                width="1000" height="<%= @article.main_image_height%>"
+                style="aspect-ratio: auto 1000 / <%= @article.main_image_height %>;"
+                width="1000" height="<%= @article.main_image_height %>"
```
Comment (klardotsh) [inline app/views/articles/show.html.erb:104]: I guess these are technically unrelated changes, and since they're whitespace-only, I'd  propose amending this commit to remove these whitespace changes (leaving behind only the 1-liner on L94 as the body of this PR).

**Review by klardotsh — COMMENTED** [review]
Hey, thanks for this contribution, and for keeping it so small and relatively easy to digest! To my eye this looks pretty good, but I mostly do infra and devtools stuff here, and so I'm not going to formally approve. That said, when you mark this draft as ready for review, two members of our team will automatically be assigned for review and I suspect it'll be a quick back-and-forth process before merge on this one.

File: app/views/articles/show.html.erb (hunk 1 above)
Comment (narender2031) [reply app/views/articles/show.html.erb:104]: @klardotsh these changes are added by the commit hook linter. But i will check again and try to remove this.

**Comment by narender2031** [comment]
> Hey, thanks for this contribution, and for keeping it so small and relatively easy to digest! To my eye this looks pretty good, but I mostly do infra and devtools stuff here, and so I'm not going to formally approve. That said, when you mark this draft as ready for review, two members of our team will automatically be assigned for review and I suspect it'll be a quick back-and-forth process before merge on this one.

Thanks for the support @klardotsh .

File: app/views/articles/show.html.erb (hunk 1 above)
Comment (klardotsh) [reply app/views/articles/show.html.erb:104]: Oh I see. If that's the case, go ahead and leave them in then, that's fine! Thanks for using the linter!

Comment (narender2031) [reply app/views/articles/show.html.erb:104]: Thanks @klardotsh

**Review by rt4914 — APPROVED** [review]
Thanks a lot for this valuable contribution @narender2031 
This may seem like a minor modification, but its significance should not be underestimated.
🚀 🎉

**Review by benhalpern — APPROVED** [review]

**Comment by narender2031** [comment]
After a long time able to contribute on this amazing product. Feeling amazed.
